Kozariz.Edelveis.EF.Database/DbContext.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Kozariz.Edelveis.Models.Configuration;
+using Kozariz.Edelveis.Models.UsersTable;
+
+namespace Kozariz.Edelveis.EF.Database
+{
+    public class MyDbContext : DbContext
+    {
+
+        public MyDbContext(DbContextOptions<MyDbContext> options) : base(options) { }
+
+        public DbSet<User> UsersTable { get; set; }
```

Comment:
ChatGPT
The code defines a MyDbContext class that inherits from DbContext and represents the database context for the application, including the UsersTable DbSet property for the User entity, but it lacks proper encapsulation by not specifying access modifiers for the class and its members, which could lead to potential issues in terms of maintainability and data security.

Reply:
Thanks for pointing that out, I will fix this later on